programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -171,27 +171,53 @@ static void on_command_resp(const tunnel_result* result, void *ctx) {
     ZITI_LOG(INFO, "resp[%d,len=%zd] = %.*s",
             result->success, json_len, (int)json_len, json, result->data);
 
+    if (result->success && result->data != NULL) {
+        tunnel_comand tnl_res_cmd = {0};
```

Comment:
I know the `tunnel_comand` model was added before this PR, but this is the first time I've noticed "command" is spelled incorrectly.